source/_posts/2018-01-16-Testing-Vuejs-in-Rails-with-Webpacker-and-Jest.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: "Testing Vue.js in Rails with Webpacker and Jest"
+category:
+comments: true
+author:
+  name: Mario Saul
+  email: mario@wyeworks.com
+  twitter_handle: mario_saul
+  github_handle:  mariiio
+  image:  /images/team/mario-saul.jpg
+  description: Software Developer at Wyeworks.
+---
+
+In the project I'm working on, I was given the task of investigating how to integrate [Vue.js](https://vuejs.org/) with our existing Rails app. So I started reading the official guide, watching tutorials and reading various posts until I finally got a full working Vue component.
```

Comment:
Maybe you could use `taking a look` instead of `reading` twice.

`fully working Vue component`?